app/org/sagebionetworks/bridge/models/accounts/Account.java

```diff
@@ -34,14 +41,91 @@ default ConsentSignature getActiveConsentSignature(SubpopulationGuid subpopGuid)
         return null;
     }
     
+    /**
+     * Gets an immutable copy of the consents this account has for the given subpopulation. Consents should be returned
+     * in the order they were signed (earliest first). Returns an empty list if there are no consents.
+     */
+    default List<ConsentSignature> getConsentSignatureHistory(SubpopulationGuid subpopGuid) {
+        List<ConsentSignature> signatures = Lists.newArrayList();
+        for (Map.Entry<HibernateAccountConsentKey, HibernateAccountConsent> entry : getConsents().entrySet()) {
```

Comment:
We have tests for getAllConsentSignatureHistories() but not for getConsentSignatureHistory(). They run similar but separate code paths. Should we have unit tests for getConsentSignatureHistory()?

Reply:
Yes definitely. Added to HibernateAccountTest.